modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase_search\ExistingSite;
+
+use Drupal\Core\Url;
+use Drupal\Tests\oe_showcase\ExistingSite\ShowcaseExistingSiteTestBase;
+
+/**
+ * Tests the OEL Search feature integration.
+ */
+class IntegrationTest extends ShowcaseExistingSiteTestBase {
+
+  /**
+   * Test the search page.
+   */
+  public function testCreateShowCasePage() {
```

Comment:
The name with Create and Page confuses me. Remove the "Create": testShowCasePage.
```suggestion
  public function testShowCasePage() {
```

Reply:
we agreed to name it "testSearchFeature"

Reply:
Right. The one thing I don't want to see in test method names is "showcase" :)